### ISSUE TITLE
[WIP] Create branch for v3 work

### ISSUE THREAD

**Issue by kadel** [redhat-developer/odo#5313]
Planned odo v3  (https://github.com/redhat-developer/odo/milestone/19) will change CLI structure and some commands will be reworked.

I'm proposing that we create new git branch called "v2", this branch will be used to save the current state of odo v2.
This branch should be after last odo v2 release (probably 2.5.0) and right before we start merging changes from [3.0](https://github.com/redhat-developer/odo/milestone/19) milestone.

odo v3 won't be released till end of the [3.0](https://github.com/redhat-developer/odo/milestone/19) milestone, at which point we will release odo 3.0.0-alpha1. 
This means that till that time we won't do our regular sprint releases unless there is a critical bug in odo v2. If that happens we will fix bug in v2 branch, and release new bugfix odo v2.5.x version. If the bug will be also present in the code for odo v3 (at that time in "main" branch) the fix will have to be ported to "main" as well.

**Comment by kadel** [comment]
Create v2 branch today
/close
